Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -35,9 +35,18 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
 
     __block id<SentrySpan> fetchSpan = nil;
     if (context.hasChanges) {
+        __block NSDictionary<NSString *, NSDictionary *> *operations = @{
+            @"INSERTED" : [self countEntities:context.insertedObjects],
+            @"UPDATED" : [self countEntities:context.updatedObjects],
+            @"DELETED" : [self countEntities:context.deletedObjects]
+        };
+
         [SentrySDK.currentHub.scope useSpan:^(id<SentrySpan> _Nullable span) {
             fetchSpan = [span startChildWithOperation:SENTRY_COREDATA_SAVE_OPERATION
-                                          description:@"Saving database"];
+                                          description:[self descriptionForOperations:operations
+                                                                           inContext:context]];
+
+            [fetchSpan setDataValue:operations forKey:@"operations"];
```

Comment:
`h`: I'm not sure what the benefit is of adding this as data on the span? Don't we have the same information already in the description?

Reply:
The data will be the same in the case of only one entity is manipulated.
If more than one entity type is edited, the description will be more generic. 
The `Data` field will have all operations in detail.

Reply:
I think it's fine to put all the detail into the description. This is how we do it also on other SDKs, see

<img width="1228" alt="Screen Shot 2022-03-17 at 09 36 37" src="https://user-images.githubusercontent.com/2443292/158769866-c6f30728-fecb-4bfd-a19e-899c219ca145.png">

Reply:
You can also fix this in a future PR if you want.

Reply:
This view that you gave an example is the expanded detail. The time graph will show much less.
IMO Is better to have a summary in description to be able to quickly understand what is going on, and then decide to open the details or not.

Of course, this is a personal preference, I can change it if you really think is for the better.

Reply:
This matches what other SDKs do. The whole SQL query is there, even when they are large. 

e.g: https://sentry.io/organizations/sentry/performance/sentry:64b5a3cff96e444dbfbd6fdfa33ae719/?project=1&query=transaction.duration%3A%3C15m&statsPeriod=24h&transaction=sentry.tasks.store.save_event&unselectedSeries=p100%28%29